third_party/lasread/tiler_las_reader.go

```diff
@@ -50,21 +51,21 @@ func (lasFileLoader *LasFileLoader) readForOctree(inSrid int, eightBitColor bool
 		return err
 	}
 	if las.fileMode != "rh" {
-		recLengths := [4][4]int{{20, 18, 19, 17}, {28, 26, 27, 25}, {26, 24, 25, 23}, {34, 32, 33, 31}}
-
-		if las.Header.PointRecordLength == recLengths[las.Header.PointFormatID][0] {
-			las.usePointIntensity = true
```

Comment:
Unfortunately point intensity and point user data are still present but optional even in LAS 1.4. 
As such I don't think we can remove this block :/